docs/resources.rst

```diff
@@ -870,10 +870,21 @@ Mostly a hook, this uses the ``Serializer`` from ``Resource._meta``.
 
 .. method:: Resource.alter_list_data_to_serialize(self, request, data)
 
-A hook to alter list data just before it gets serialized & sent to the user.
+A hook to alter list data just before it gets serialized & sent to the user. 
 
 Useful for restructuring/renaming aspects of the what's going to be
-sent.
+sent. Occurs after any dehydration has by applied.
+
+As such this is a useful place to apply modifications which affect many
+list elements.
+
+Example:
+
+    def alter_list_data_to_serialize(self, request, data):
+        bar = some_expensive_call()
+        for obj in data['objects']:
+            obj.foo = bar
```

Comment:
This line is triggering a warning about indentation:
`/home/travis/build/django-tastypie/django-tastypie/docs/resources.rst:886:Unexpected indentation.`

See:
https://travis-ci.org/django-tastypie/django-tastypie/jobs/320152115

Maybe just a matter of mixing tabs and spaces?